contributors.js

```diff
@@ -21,4 +21,5 @@ const contributors = [
   { username: 'JayShukla8', message: 'Autumn\'s here!' },
   { username: 'SameerKhans13', message: 'Happy Hacktoberfest-24 to all!' },
   { username: 'rishi-harti768', message: 'Happy Hacktoberfest guys! cheers'}
```

Comment:
Hello @Nipun899 , please add a comma at the end of the line 23. The structure of this json file is not valid.